### PR TITLE
Rest of react query implementation

### PR DESCRIPTION
**Pull request by emilianoyaryura**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

## -

- **How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/www/pages/dashboard/developers/webhooks/[id].tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ const ApiKeys = () => {
                             setDeleting(true);
                             await deleteWebhook(data.id);
                             setDeleting(false);
-                            revalidate();
+                            queryClient.invalidateQueries();
```
Comment (julianbenegas) [inline packages/www/pages/dashboard/developers/webhooks/[id].tsx]: we don't want to invalidate ALL queries across the app. We should pass the queryKey to `invalidateQueries`

Comment (julianbenegas) [reply packages/www/pages/dashboard/developers/webhooks/[id].tsx]: Also, we should `await` this, if `invalidateQueries` returns a promise.

File: packages/www/pages/dashboard/developers/webhooks/[id].tsx [hunk 2]
```diff
@@ -161,7 +161,7 @@ const ApiKeys = () => {
                         name: name ? name : data.name,
                         url: url ? url : data.url,
                       });
-                      await revalidate();
+                      await queryClient.invalidateQueries();
```
Comment (julianbenegas) [inline packages/www/pages/dashboard/developers/webhooks/[id].tsx]: same: missing queryKey

File: packages/www/pages/dashboard/streams/[id]/index.tsx [hunk 3]
```diff
@@ -343,17 +345,26 @@ const ID = () => {
                   <DropdownMenuGroup>
                     <Record
                       stream={stream}
-                      revalidate={revalidate}
+                      revalidate={queryClient.invalidateQueries()}
```
Comment (julianbenegas) [inline packages/www/pages/dashboard/streams/[id]/index.tsx]: We want to pass the `invalidateQueries` method down, but _not_ execute it...

This is executing `invalidateQueries` and passing down whatever it gets in return.

Comment (julianbenegas) [reply packages/www/pages/dashboard/streams/[id]/index.tsx]: Also, same as comments above: we don't want to invalidate ALL the queries.

**Review by julianbenegas — CHANGES_REQUESTED** [review]

**Review by adamsoffer — APPROVED** [review]
LGTM!

**Pull request merged**